docs/how-to/monitor-your-backups.md

```diff
@@ -290,6 +291,75 @@ If you have any issues with the integration, [please contact
 us](https://torsion.org/borgmatic/#support-and-contributing).
 
 
+## Pushover hook
+
+<span class="minilink minilink-addedin">New in version 1.9.0</span>
```

Comment:
This will be 1.9.1 or later, but I can always set that after merging.

Reply:
https://github.com/borgmatic-collective/borgmatic/commit/a4fabb85217e6e57007f32248a5e06eace62f048